docs/quickstart.rst

```diff
@@ -110,13 +112,16 @@ Using `HTTPie <http://httpie.org>`_, it is as easy as:
 
 ::
 
-    $ http -v http://localhost:8000/v0/ --auth user:pass
+    $ http -v http://localhost:6543/v0/ --auth user:pass
 
+.. note::
+
+    In the case of *Basic Auth*, there is no need of registering a user/password. Just pick any combination you want, and include them in each request.
 
 Define resources
 ================
 
-In order to define a resource, just inherit from :class:`cliquet.resource.BaseResource`,
+In order to define a resource, define a class inheriting from :class:`cliquet.resource.BaseResource`,
```

Comment:
That's two times "define". Maybe "in order to define a resource, inherit from [...] in a subclass"